packages/editor/src/components/post-title/index.native.js

```diff
@@ -26,13 +26,19 @@ import { pasteHandler } from '@wordpress/blocks';
 import styles from './style.scss';
 
 class PostTitle extends Component {
+	constructor( props ) {
```

Comment:
Changes in that file look like something that could be done in separate PR. I know that probably you fixed it during work on social links but i strongly recommend split PRs to smaller. Thanks to that it would be easier to review and test :)

Reply:
Moved to the separate PR: https://github.com/WordPress/gutenberg/pull/23683